src/webapp/pages/form-page/disease-outbreak-event/utils/mapEntityToInitialFormState.ts

```diff
@@ -0,0 +1,655 @@
+                                    "appropriateCaseManagementDate",
+                                    diseaseOutbreakEventFieldIds
+                                ),
+                            },
+                        ],
+                    },
+                    {
+                        id: "5. Initiate appropriate public health countermeasures in affected communities.",
+                        title: "5. Initiate appropriate public health countermeasures in affected communities.",
+                        isVisible: true,
+                        required: true,
+                        fields: [
+                            {
+                                id: getFieldIdFromIdsDictionary(
+                                    "initiatePublicHealthCounterMeasuresDate",
+                                    diseaseOutbreakEventFieldIds
+                                ),
+                                label: "Date Completed",
+                                isVisible: true,
+                                errors: [],
+                                type: "date",
+                                value:
+                                    diseaseOutbreakEvent?.earlyResponseActions
+                                        .initiatePublicHealthCounterMeasures.date || null,
+                                width: "200px",
+                                hasNotApplicable: true,
+                                required: true,
+                                showIsRequired: false,
+                                disabled:
+                                    diseaseOutbreakEvent?.earlyResponseActions
+                                        .initiatePublicHealthCounterMeasures.na,
+                            },
+                            {
+                                label: "N/A",
+                                id: getFieldIdFromIdsDictionary(
+                                    "initiatePublicHealthCounterMeasuresNA",
+                                    diseaseOutbreakEventFieldIds
+                                ),
+                                isVisible: true,
+                                errors: [],
+                                type: "boolean",
+                                value:
+                                    diseaseOutbreakEvent?.earlyResponseActions
+                                        .initiatePublicHealthCounterMeasures.na || false,
+                                width: "65px",
+                                notApplicableFieldId: getFieldIdFromIdsDictionary(
+                                    "initiatePublicHealthCounterMeasuresDate",
+                                    diseaseOutbreakEventFieldIds
+                                ),
+                            },
+                        ],
+                    },
+                    {
+                        id: "6. Initiate appropriate risk communication and community engagement activities.",
```

Comment:
[just a note, no need to change] to make reordering easier, the index may be added by some parent function.